docs/docsite/rst/playbooks_reuse_includes.rst

```diff
@@ -16,13 +16,13 @@ Please refer to  :doc:`playbooks_reuse` for documentation concerning the trade-o
 Importing Playbooks
 ```````````````````
 
-It is possible to include playbooks inside a master playbook. For example::
+It is possible to import playbooks inside a master playbook. For example::
 
     ---
-    import_playbook: webservers.yml
-    import_playbook: databases.yml
+    - import_playbook: webservers.yml
+    - import_playbook: databases.yml
 
-Each playbook listed will be run in the order they are listed.
+Each playbook listed will be imported in the order they are listed.  (import_playbook was added in 2.4)
```

Comment:
@bcoca I'm happy to turn it over to you.   As an aside, I think that PlaybookInclude.load should check for an entry that has both a playbook_import: and include: entry, though it's pretty much of an edge case.

    - import_playbook: play1.yml
      include: play2.yml  # Currently this invalid syntax silently "works" but fails to import play1.  Pretty obscure though.